parser.y

```diff
@@ -5564,6 +5580,12 @@ InsertValues:
 	{
 		$$ = &ast.InsertStmt{Setlist: $2.([]*ast.Assignment)}
 	}
+|	"ROW" '(' ValuesList ')'
+	{
+		$$ = &ast.InsertStmt{
+			Lists: $3.([][]ast.ExprNode),
+		}
+	}
```

Comment:
`INSERT INTO x ROW (...)` is not valid. should be changing `ValuesList` instead.

Reply:
fix